SimplenoteScreenshots/SimplenoteScreenshots.swift

```diff
@@ -30,69 +30,24 @@ class SimplenoteScreenshots: XCTestCase {
 
         let password = app.secureTextFields["Password"]
         XCTAssertTrue(password.waitForExistence(timeout: 10))
-        // -.-'
-        password.tap()
-        password.pasteText(text: ScreenshotsCredentials.testUserPassword)
+        password.typeSecureText(ScreenshotsCredentials.testUserPassword, using: app)
```

Comment:
In hindsight, that comment wasn't that useful. Extracting this unexpected logic into a dedicated function is a better option.